### PR TITLE
Update changelog after NNS-Dapp release

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation

A release has been [deployed to production](https://dashboard.internetcomputer.org/proposal/125318).

# Changes

- Changelog - split out the changes included in the release.
- Increment the patch version of the nns-dapp.

### CODE REVIEW

File: CHANGELOG-Nns-Dapp-unreleased.md [hunk 1]
```diff
@@ -15,44 +15,30 @@ proposal is successful, the changes it released will be moved from this file to
...
-* Fix UI bug when commitment was very low.
 * Remove robots metatag to allow search engines to crawl NNS Dapp.
 
 #### Security
 
 #### Not Published
 
-* New feature flag `ENABLE_MY_TOKENS`.
-* Detailed values of the Neurons' Fund and direct participation in the project detail page.
 * Added `approveTransfer` in icrc-ledger API.
```
Comment (bitdivine) [inline CHANGELOG-Nns-Dapp-unreleased.md:32]: This line needs to be deleted.

Comment (bitdivine) [reply CHANGELOG-Nns-Dapp-unreleased.md]: Done.

**Review by lmuntaner — APPROVED** [review]
LGTM!

File: CHANGELOG-Nns-Dapp.md [hunk 2]
```diff
@@ -11,6 +11,42 @@ The NNS Dapp is released through proposals in the Network Nervous System. Theref
 Unreleased changes are added to `CHANGELOG-Nns-Dapp-unreleased.md` and moved
 here after a successful release.
 
+## Proposal 125318
+
+### Application
+
+#### Added
+
+* Added `approveTransfer` in icrc-ledger API.
```
Comment (dskloetd) [inline CHANGELOG-Nns-Dapp.md:20]: Would be slightly better to have this under `#### Not Published`.

Comment (bitdivine) [reply CHANGELOG-Nns-Dapp.md]: Done

File: frontend/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dfinity/nns-dapp",
-  "version": "2.0.52",
+  "version": "2.0.53",
```
Comment (dskloetd) [inline frontend/package.json:3]: Are the cargo patch version and frontend patch version not supposed to be in sync?

Comment (lmuntaner) [reply frontend/package.json:3]: Yes, I must have messed it up in my previous PR. @bitdivine can you sync them?

Comment (bitdivine) [reply frontend/package.json:3]: As 53 I presume...

Comment (bitdivine) [reply frontend/package.json:3]: Done.  If they are meant to be in sync, should there be a CI check to that effect?  I assume so....

Comment (bitdivine) [reply frontend/package.json:3]: Please check, @dskloetd , and mark this conversation as "resolved" if the patch number is as you expect.

**Review by dskloetd — APPROVED** [review]

**Pull request merged**